homeassistant/components/netatmo/camera.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         self._camera = cast(NaModules.Camera, netatmo_device.device)
         self._id = self._camera.entity_id
         self._home_id = self._camera.home.entity_id
+        self._attr_name = self._camera.name
```

Comment:
Does the camera have a device attached? If the device name is the camera name, please set `_attr_name = None` as it will then use the device name as entity name

Reply:
The device name _is_ the camera name. On checking, this explains why the name was duplicating. Thanks for the tip - I am testing it now.

Reply:
Unfortunately, that did not work - back to showing a null value in the Camera screen. This is what I put:

```
self._camera = cast(NaModules.Camera, netatmo_device.device)
self._id = self._camera.entity_id
self._home_id = self._camera.home.entity_id
self._attr_name = None
self._device_name = self._camera.name
self._model = self._camera.device_type
self._config_url = CONF_URL_SECURITY
self._attr_unique_id = f"{self._id}-{self._model}"
self._quality = DEFAULT_QUALITY
self._monitoring: bool | None = None
self._light_state = None
```

Reply:
_attr_name is set to None in the Netatmo base class.

Reply:
I mean, if I understand correctly, the name duplicating is wrong.

But isn't this just a bug in the media browser then instead of a bug in the entity name?

Reply:
Exactly my understanding.